tests/utils/test-helpers.ts

```diff
@@ -52,6 +52,11 @@ import { AddressStxBalance } from '../../src/api/schemas/entities/addresses';
 import { ServerStatusResponse } from '../../src/api/schemas/responses/responses';
 
 export async function migrate(direction: 'up' | 'down') {
+  const connArgs = await createSchema();
```

Comment:
It would be cleaner if you separate the schema creation from the connection arguments
```suggestion
  const connArgs = getConnectionArgs();
  await createSchema(connArgs);
```